build.sh

```diff
@@ -281,8 +324,9 @@ function main {
     # build_all.
     if [ -z ${1+x} ]; then
         # In case we don't have any subcommand then
-        # we want to build all.
+        # we want to build and run all.
         build_all
+        run_all
     elif [ "$1" == "all" ]; then
         build_all
     elif [ "$1" == "misses_image" ]; then
```

Comment:
maybe we can also add an `run` sub command:

```suggestion
    elif [ "$1" == "run" ]; then
        run_all
    elif [ "$1" == "misses_image" ]; then
```

Reply:
Adjusted!